src/api/app/models/user.rb

```diff
@@ -75,7 +75,7 @@ class User < ApplicationRecord
             uniqueness: { message: 'is the name of an already existing user' }
 
   validates :login,
-            format: { with: %r{\A[\w \$\^\-\.#\*\+&'"]*\z},
+            format: { with: %r{\A[\w\$\^\-\.#\*\+&'"]*\z},
                       message: 'must not contain invalid characters' }
```

Comment:
Could we improve this message by including the invalid characters? This way, users know what to change. Otherwise, it's just a shot in the dark.

Reply:
or have the same pattern on the HTML input?
https://www.w3schools.com/tags/att_input_pattern.asp

Reply:
or both :rofl: I do it :+1: